assets/js/main.js

```diff
@@ -1,40 +1,64 @@
+/* eslint-disable max-classes-per-file */
+
+// Class constructor
+class Book {
+  constructor(title, author) {
+    this.title = title;
+    this.author = author;
+  }
+}
+
+class BooksCollection {
+  constructor() {
+    this.booksArray = [];
+  }
+
+  addBookToCollection = (title, author) => {
+    const newBook = new Book(title, author);
+    this.booksArray.push(newBook);
+  };
+
+  removeBookFromCollection = (title) => {
```

Comment:
Great work with the remove method. :clap:  It successfully handles duplicated records.